.github/CODEOWNERS

```
@@ -0,0 +1,233 @@
+/helmfile.d/values/ingress-nginx.yaml.gotmpl @elastisys/goto-ingress/auth
+/helmfile.d/values/kubeapi-metrics.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/kube-prometheus-stack-* @elastisys/goto-monitoring-stack
+/helmfile.d/values/kube-state-metrics-extra-resource-metrics.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/kured.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/letsencrypt.yaml.gotmpl @elastisys/goto-ingress/auth
+/helmfile.d/values/metrics-server.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/minio.yaml.gotmpl @elastisys/goto-block-storage
+/helmfile.d/values/node-local-dns.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/openstack-monitoring.yaml.gotmpl @elastisys/goto-logging-stack
+/helmfile.d/values/prometheus-alerts-sc.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/prometheus-blackbox-exporter-* @elastisys/goto-monitoring-stack
+/helmfile.d/values/prometheus-opensearch-exporter.yaml.gotmpl @elastisys/goto-logging-stack
+/helmfile.d/values/prometheus-user-alerts-wc.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/s3-exporter.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/sc-servicemonitor.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/tekton.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/user-alertmanager.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/user-rbac.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/velero-* @elastisys/goto-security
+/helmfile.d/values/wc-servicemonitor.yaml.gotmpl @elastisys/goto-monitoring-stack
+
+#
+# Skipped files in helmfile.d/values
+# admin-namespaces-sc.yaml.gotmpl
+# admin-namespaces-wc.yaml.gotmpl
+#
+
+/helmfile.d/README.md @elastisys/goto-scripts
+/helmfile.d/state.yaml @elastisys/goto-scripts
+
+#
+# images
+#
+
+/images/backup-postgres/ @elastisys/goto-postgresql
+/images/calico-accountant/ @elastisys/goto-kubernetes
+/images/elasticsearch-curator/ @elastisys/goto-logging-stack
+/images/log-manager/ @elastisys/goto-logging-stack
+/images/rclone-sync/ @elastisys/goto-security
+
+#
+# Skipped files in images
+# curl-jq/
+#
+
+/migration/ @elastisys/goto-pipeline/qa/release
+
+/pipeline/ @elastisys/goto-pipeline/qa/release
+
+/release/ @elastisys/goto-pipeline/qa/release
+
+/roles/ @elastisys/goto-scripts
+
+#
+# scripts
+#
+
+/scripts/azure/ @elastisys/goto-azure
+/scripts/bom/ @elastisys/goto-pipeline/qa/release
+/scripts/grafana-dashboards @elastisys/goto-monitoring-stack
+/scripts/local-clusters/ @elastisys/goto-scripts
+/scripts/migration/ @elastisys/goto-pipeline/qa/release
+/scripts/report/ @elastisys/goto-security
+/scripts/requirements/ @elastisys/goto-scripts
+/scripts/restore/ @elastisys/goto-container-registry
+/scripts/S3/ @elastisys/goto-scripts
+/scripts/sbom/ @elastisys/goto-pipeline/qa/release
+/scripts/* @elastisys/goto-scripts
```

Could this be changed to
```suggestion
/scripts/ @elastisys/goto-scripts
```

---

It will give a bit of a different meaning, check my comment to your previous comment.

---

You should be able to set us for `/scripts/` before other entries in script, and the more specific ones will override the generic one.

From the docs:
```
# In this example, @octocat owns any file in the `/apps`
# directory in the root of your repository except for the `/apps/github`
# subdirectory, as this subdirectory has its own owner @doctocat
/apps/ @octocat
/apps/github @doctocat
```